automation/taskcluster/lib/tasks.py

```diff
@@ -269,7 +270,7 @@ def _craft_build_ish_task(
         }
 
         return self._craft_default_task_definition(
-            'mobile-1-b-fenix' if is_staging else 'mobile-3-b-fenix',
+            'mobile-{}-b-fenix'.format(self.trust_level),
```

Comment:
Heads up that I don't think that `is_staging` is used in this function any more :)

Reply:
Oh good call. Fixed!